src/validations/rules/ssp.sch

```diff
@@ -747,6 +747,22 @@
                     else
                         (true())">Implemented Requirement <xsl:value-of
                     select="@control-id" /> exists.</sch:assert>
+            <sch:assert
+                diagnostics="configuration-management-controls-described-diagnostic"
+                id="configuration-management-controls-described"
+                role="warning"
+                test="
+                    if (matches(@control-id, 'cm-9'))
+                    then
+                        (if (self::oscal:implemented-requirement//*[matches(., 'CIS|Center for Internet Security|SCAP')])
+                        then
+                            (true())
+                        else
+                            (false()))
+                    else
+                        (true())">Implemented Requirement <xsl:value-of
+                    select="@control-id" /> exists and has acceptable keywords.</sch:assert>
```

Comment:
I checked and there is no relevant documentation, so issue #313 is the only documentation and implies this is FedRAMP-specific (a proper treatment would reference the [National Checklist Program](https://ncp.nist.gov/general), but we can leave that for the RMF/FedRAMP split). So please add `fedramp:specific="true"` and a `see` attribute referencing issue #313  so some future puzzled person will have a reference to an otherwise undocumented request.